src/agent0/ethpy/base/transactions.py

```diff
@@ -533,6 +545,10 @@ async def async_smart_contract_transact(
         The value field to set for transaction options.
     txn_options_gas : int | None
         The gas field to set for transaction options.
+    txn_options_base_fee_multiple: float | None = None
+        The multiple by which to multiply the base fee to set for transaction options. Defaults to 1.
```

Comment:
this is a weird phrasing. maybe something  like "the base fee is multiplied by this before it is added to the max priority fee to set gas"?

Reply:
improved in24d151d63fc5213669327255444ce5c09981bfa8